.github/actions/pkg-check/action.yml

```
@@ -33,9 +33,9 @@ runs:
 
   - name: Check package
     run: |
+      git diff --exit-code || exit $?  # make sure there are no local unstaged changes
```

nit: The command exiting with non-zero status won't make the CI fail?
```suggestion
      git diff --exit-code  # make sure there are no local unstaged changes
```

---

That's why I added it. To have it fail if local changes are introduced